xyzspaces/apis.py

```diff
@@ -513,12 +513,18 @@ def delete_space(self, space_id: str) -> str:
     # Read Features
 
     def get_space_features(
-        self, space_id: str, feature_ids: List[str]
+        self,
+        space_id: str,
+        feature_ids: List[str],
+        force_2d: Optional[bool] = None,
     ) -> dict:
         """Get features by ID.
 
         :param space_id: The desired space ID.
         :param feature_ids: A list of feature_ids.
+        :param force_2d: If set to True the features in the response
+            will have only X's and Y's as coordinates,by default all
```

Comment:
Would prefer "will have only X and Y components, by default all" instead of "will have only X's and Y's as coordinates,by default all" everywhere.